examples/Cargo.toml

```diff
@@ -0,0 +1,50 @@
+[workspace]
+members = [
+  "abitest/abitest_common",
+  "abitest/module_0/rust",
+  "abitest/module_1/rust",
+  "abitest/tests",
+  "aggregator/backend",
+  "aggregator/common",
+  "aggregator/module/rust",
+  "authentication/client",
+  "chat/module/rust",
+  "hello_world/module/rust",
+  "machine_learning/module/rust",
+  "private_set_intersection/module/rust",
+  "running_average/module/rust",
+  "rustfmt/module/rust",
+  "translator/common",
+  "translator/module/rust",
+]
+
+# Patch dependencies on oak crates so that they refer to the versions within this same repository.
+#
+# From https://doc.rust-lang.org/book/ch14-03-cargo-workspaces.html :
+# > Cargo doesn’t assume that crates in a workspace will depend on each other, so we need to be
+# > explicit about the dependency relationships between the crates.
+[patch.crates-io]
+# Oak.
+oak = { path = "../sdk/rust/oak" }
+oak_abi = { path = "../oak_abi" }
+oak_runtime = { path = "../oak/server/rust/oak_runtime" }
+oak_tests = { path = "../sdk/rust/oak_tests" }
+oak_utils = { path = "../oak_utils" }
+# Examples.
+abitest_common = { path = "abitest/abitest_common" }
+abitest_0_frontend = { path = "abitest/module_0/rust" }
+abitest_1_backend = { path = "abitest/module_1/rust" }
+abitest_tests = { path = "abitest/tests" }
+aggregator = { path = "aggregator/module/rust" }
+aggregator_backend = { path = "aggregator/backend" }
+aggregator_common = { path = "aggregator/common" }
+chat = { path = "chat/module/rust" }
+hello_world = { path = "hello_world/module/rust" }
+machine_learning = { path = "machine_learning/module/rust" }
+private_set_intersection = { path = "private_set_intersection/module/rust" }
+running_average = { path = "running_average/module/rust" }
+rustfmt = { path = "rustfmt/module/rust" }
+translator = { path = "translator/module/rust" }
+translator_common = { path = "translator/common" }
```

Comment:
Why are all these patches needed?

Reply:
I guess Cargo needs these to find the dependencies. Previously, they were in the top-level `Cargo.toml`.

Reply:
I'm aware they were there, but I don't think they are actually relevant any more. I think they were just a convenience so that we could use `--package=translator`, but now that they are in a separate workspace we probably need to specify the entire manifest path anyways, so I don't think they buy us anything. Or it would be good to know if they are actually used for anything else :)

Reply:
I see! I removed them except from `translator_common = { path = "translator/common" }`. That one is still needed, since it is a dependency for `translator` and `hello-world`. Specifying the manifest-path does not seem to bring `translator_common` into scope.